src/main/java/stsjorbsmod/actions/ToilAction.java

```diff
@@ -0,0 +1,61 @@
+package stsjorbsmod.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.utility.WaitAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.vfx.UpgradeShineEffect;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardBrieflyEffect;
+import com.megacrit.cardcrawl.vfx.combat.FlashAtkImgEffect;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class ToilAction extends AbstractGameAction {
+    private DamageInfo info;
+    private AbstractCard theCard = null;
+
+    public ToilAction(AbstractCreature target, DamageInfo info) {
+        this.info = info;
+        setValues(target, info);
+        actionType = AbstractGameAction.ActionType.DAMAGE;
+        duration = Settings.ACTION_DUR_MED;
+    }
+
+    public void update() {
+        if (duration == Settings.ACTION_DUR_MED && target != null) {
+            AbstractDungeon.effectList.add(new FlashAtkImgEffect(target.hb.cX, target.hb.cY, AbstractGameAction.AttackEffect.NONE));
```

Comment:
Is this intended to be using `AttackEffect.NONE`?

Reply:
nope